content/03.ml-concepts.md

```diff
@@ -1,24 +1,29 @@
-## Tip 1: Concepts that apply to machine learning also apply to deep learning {#concepts}
-Deep learning is a distinct subfield of machine learning, but it is still a subfield.
-DL has proven to be an extremely powerful paradigm capable of outperforming “traditional” machine learning approaches in certain contexts, but it is not immune to the many limitations inherent to machine learning.
-Many best practices for machine learning also apply to deep learning.
-Like all computational methods, deep learning should be applied in a systematic manner that is reproducible and rigorously tested.
+## Tip 1: Adhere to machine learning best practices for rigorous model evaluation {#concepts}
 
-Those developing deep learning models should select data that are relevant to the problem at hand; non-salient data can hamper performance or lead to spurious conclusions.
-Biases in testing data can also unduly influence measures of model performance, and it may be difficult to directly identify confounders from the model.
-Investigators should consider the extent to which the outcome of interest is likely to be predictable from the input data and begin by throughly inspecting the input data.
-Suppose that there are robust heritability estimates for a phenotype that suggest that the genetic contribution is modest but a deep learning model predicts the phenotype with very high accuracy.
+
+In recent years, deep learning has proven to be an extremely powerful paradigm capable of outperforming "traditional" or "conventional" machine learning approaches.
+The recent success behind applying deep learning is partly owed to the fact that DL algorithms are less dependent on manual feature extraction and are capable of achieving high predictive performance when operating on raw-data, such as images and text directly.
+In contrast, traditional machine learning approaches often require manual feature extraction to produce acceptable results. 
+However, the ability of deep learning to perform implicit feature extraction can be regarded as a double-edged sword.
+For instance, those developing deep learning models not only need to ensure that the data contains information relevant to the problem at hand but is also free of systematic biases.
```

Comment:
It seems like this sentence is suggesting that you don't have to make sure your data is "free of systematic biases" if you're not using DL.

Reply:
Yeah, it could sound that way ... the point was more like to provide some extra motivation why this is even more important for DL than for ML, as there is less manual feature engineering involved, which kind of reduces bias (e..g, if I extract facial keypoints and train a classifier, this classifier will obviously not sensitive to skin color or background noise -- the facial keypoint extractor might, but that's a different kind of beast and way too general for an article that aims to discuss deep learning, not computational research in general, in my opinion; i.e., I assume that most people worked with datasets before and read this article primarily because they are interested in DL).

Reply:
Doesn't that make more sense in one of the sections devoted entirely to overfitting or baselines?